app/src/amazonWebview/java/org/mozilla/focus/iwebview/WebViewProvider.kt

```diff
@@ -67,6 +67,8 @@ private fun initWebSettings(context: Context, settings: AmazonWebSettings) = wit
     javaScriptEnabled = true
     domStorageEnabled = true
 
+    mediaPlaybackRequiresUserGesture = false // Allow YouTube to auto-play.
```

Comment:
Nit: I'd say "Allows auto-play" - unless you really want to emphasize Youtube with something like "Allows auto-play (which improves YouTube experience)"